source/jormungandr/jormungandr/realtime_schedule/cleverage.py

```diff
@@ -145,7 +145,7 @@ def _get_next_passage_for_route_point(self, route_point, count=None, from_dt=Non
         return self._get_passages(route_point, r.json())
 
     def status(self):
-        return {'id': self.rt_system_id,
+        return {'id': unicode(self.rt_system_id),
```

Comment:
@kinnou02 changed it in status also... Can be reverted if not smart (like we want to search from it in redis where we don't use unicode...)

Reply:
?

Reply:
OK, stupid remark, we don't care :smile: